packages/number-field/src/NumberField.ts

```diff
@@ -160,6 +160,12 @@ export class NumberField extends TextfieldBase {
         }
         const oldValue = this._value;
         this._value = value;
+        if (!this.managedInput && this.lastCommitedValue != this.value) {
```

Comment:
`!=` allows coercion, where at `!==` does not. I'm not sure we'd ever be in a situation for coercion to occur here, but if it's possible to use instead, let's normalize on `!==` when possible.

Reply:
Also, will this dispatch a `change` event if the use use `el.value = 'new value'` (but a valid value) from the outside?

Reply:
If this doesn't do that, we should add a test to confirm for the future. If it does do that, we should add a test to confirm that the alternate approach that you take to get there, too.

Reply:
thnx for noticing this! updated it🥂